d2core/d2map/d2mapgen/act1_overworld.go

```diff
@@ -64,8 +63,6 @@ func GenerateAct1Overworld(mapEngine *d2mapengine.MapEngine) {
 		// North Exit
 		mapEngine.PlaceStamp(townStamp, mapWidth-townSize.Width, mapHeight-townSize.Height)
 	}
-
-	mapEngine.RegenerateWalkPaths()
```

Comment:
Could removing this cause issue with walking in the game?

Reply:
Walking is not working after removing walkmesh, I was thinking of implementing it in a separate PR but I think @essial  will work on that and I will sync with his changes after that.